examples/mutate/abstract functionality blueprint/pipe_output.py

```diff
@@ -6,43 +6,43 @@
 
 
 # data1 and data2
-def _filter(some_data: pd.DataFrame) -> pd.DataFrame:
+def filter_(some_data: pd.DataFrame) -> pd.DataFrame:
     return some_data.dropna()
 
 
 # data 2
 # this is for value
-def _add_missing_value(some_data: pd.DataFrame, missing_row: List[Any]) -> pd.DataFrame:
+def add_missing_value_(some_data: pd.DataFrame, missing_row: List[Any]) -> pd.DataFrame:
```

Comment:
We should only have `_` at the end of of fns if they're builtins (E.G. `filter`) -- otherwise just normal names